Realm/ObjectServerTests/SwiftObjectServerTests.swift

```diff
@@ -31,522 +31,522 @@ class Person: Object {
-        let config = user.configuration(realmURL: URL(string: "realms://localhost:9443/~/default"),
-                                        serverValidationPolicy: .system)
-
-        let ex = expectation(description: "Waiting for error handler to be called")
-        SyncManager.shared.errorHandler = { (error, session) in
-            ex.fulfill()
-        }
-
-        _ = try! Realm(configuration: config)
-        self.waitForExpectations(timeout: 4.0)
-    }
-
-    func testSecureConnectionToLocalhostWithValidationDisabled() {
-        let user = try! synchronouslyLogInUser(for: basicCredentials(), server: authURL)
-        let config = user.configuration(realmURL: URL(string: "realms://localhost:9443/~/default"),
-                                        serverValidationPolicy: .none)
-        SyncManager.shared.errorHandler = { (error, session) in
-            XCTFail("Unexpected connection failure: \(error)")
-        }
-
-        let realm = try! Realm(configuration: config)
-        self.waitForUploads(for: realm)
-    }
-
-    func testSecureConnectionToLocalhostWithPinnedCertificate() {
-        let user = try! synchronouslyLogInUser(for: basicCredentials(), server: authURL)
-        let certURL = URL(string: #file)!
-            .deletingLastPathComponent()
-            .appendingPathComponent("certificates")
-            .appendingPathComponent("localhost.cer")
-
-        let config = user.configuration(realmURL: URL(string: "realms://localhost:9443/~/default"),
-                                        serverValidationPolicy: .pinCertificate(path: certURL))
-        SyncManager.shared.errorHandler = { (error, session) in
-            XCTFail("Unexpected connection failure: \(error)")
-        }
-
-        let realm = try! Realm(configuration: config)
-        self.waitForUploads(for: realm)
-    }
-
-    func testSecureConnectionToLocalhostWithIncorrectPinnedCertificate() {
-        let user = try! synchronouslyLogInUser(for: basicCredentials(), server: authURL)
-        let certURL = URL(string: #file)!
-            .deletingLastPathComponent()
-            .appendingPathComponent("certificates")
-            .appendingPathComponent("localhost-other.cer")
-        let config = user.configuration(realmURL: URL(string: "realms://localhost:9443/~/default"),
-                                        serverValidationPolicy: .pinCertificate(path: certURL))
-
-        let ex = expectation(description: "Waiting for error handler to be called")
-        SyncManager.shared.errorHandler = { (error, session) in
-            ex.fulfill()
-        }
-
-        _ = try! Realm(configuration: config)
-        self.waitForExpectations(timeout: 4.0)
-    }
-
-    private func realmURLForFile(_ fileName: String) -> URL {
-        let testDir = RLMRealmPathForFile("realm-object-server")
-        let directory = URL(fileURLWithPath: testDir, isDirectory: true)
-        return directory.appendingPathComponent(fileName, isDirectory: false)
-    }
+//
```

Comment:
Disabling code by commenting out each line separately makes the diffs a mess. Doing it via `#if false` makes it so that when the code is reenabled later it's actually possible to see what's changing to make it work again.